command/oauth/cmd.go

```diff
@@ -784,18 +784,24 @@ func (o *oauth) DoLoopbackAuthorization() (*token, error) {
 		return nil, err
 	}
 
-	if err := exec.OpenInBrowser(authURL, o.browser); err != nil {
-		fmt.Fprintln(os.Stderr, "Cannot open a web browser on your platform.")
-		fmt.Fprintln(os.Stderr)
-		fmt.Fprintln(os.Stderr, "Open a local web browser and visit:")
-		fmt.Fprintln(os.Stderr)
-		fmt.Fprintln(os.Stderr, authURL)
-		fmt.Fprintln(os.Stderr)
+	skipBrowser := os.Getenv("STEP_OPEN_BROWSER") == "false"
+
+	if !skipBrowser {
+			if err := exec.OpenInBrowser(authURL, o.browser); err != nil {
+					fmt.Fprintln(os.Stderr, "Cannot open a web browser on your platform.")
```

Comment:
Don't apply this directly, but can you make it look like this:
```suggestion
	if skipBrowser := os.Getenv("STEP_OPEN_BROWSER") == "false"; skipBrowser {
			// no browser case
	} else {
	                // browser case
	}
```

We don't have many boolean env vars, but for the ones that do exist, we use numeric values 1/0 to toggle the behavior. That should be consistent. We could support both numeric values and strings, but then it should go into a helper func; potentially even in our `cli-utils` package.

There's an existing env var `STEP_INTEGRATION_DEBUG`, which seems to exist for testing purposes. This _could_ become `TEST_INTEGRATION_OPEN_BROWSER` to group that together.

Reply:
I can see use cases where people might want this URL like I do that are not using them for integration tests. I'll leave it as is for now. We can leave it undocumented for the time being to see if we want to change it later.